src/main/java/uk/gov/hmcts/reform/sscs/model/HearingLocations.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.sscs.exception.ValidationError;
+import uk.gov.hmcts.reform.sscs.validator.EnumPattern;
+
+import javax.validation.constraints.NotEmpty;
+
+@NoArgsConstructor
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class HearingLocations {
+
+    @NotEmpty(message = ValidationError.LOCATION_TYPE_EMPTY)
+    private String locationType;
+
+    @EnumPattern(enumClass = LocationType.class, fieldName = "locationId")
```

Comment:
Shouldn't this enum be on location type?

Reply:
I checked tribunals api also separate it to another class. Actually most of the Enum created by @dholaramgurjar . Probably update from his repo would be better